package.json

```diff
@@ -7,12 +7,13 @@
   "scripts": {
     "clean": "rimraf build",
     "build": "npm run clean && tsc -p ./tsconfig.json && tsc -p ./examples/tsconfig.json",
-    "prepare": "npm run build && npm run test",
+    "prepack": "npm run build && npm run test",
```

Comment:
I changed this task here because `prepare` was also run after a local `npm install` - this was nice for local development, but also meant that every CI run would run the tests twice. Now this only runs before pack/publish.